test/property_test/gradualizer_erlang_abstract_code.erl

```diff
@@ -0,0 +1,3117 @@
+%%% This file is part of PropEr.
```

Comment:
Good points about the abstract code and this proper file. I think it would be good to add a note about this in the LICENCE file. (Not necessarily in this PR.)